config/settings/production.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+DEBUG = False
+
+ALLOWED_HOSTS = os.getenv('ALLOWED_HOSTS').split(',') \
```

Comment:
`ALLOWED_HOSTS` should always been specified in production. If they are not, just crash.
```suggestion
ALLOWED_HOSTS = os.environ['ALLOWED_HOSTS'].split(',')
```